test/e2e/performanceprofile/functests/1_performance/performance.go

```diff
@@ -282,6 +283,39 @@ var _ = Describe("[rfe_id:27368][performance]", Ordered, func() {
 		})
 	})
 
+	Context("Using performance profile", func() {
+		It("Should have system services running on the system.slice cgroup", func() {
+			for _, node := range workerRTNodes {
+				processesFound := make([]string, 0)
+				rootCgroupPath := "/rootfs/sys/fs/cgroup/cpuset/cgroup.procs"
+				isV2, err := cgroup.IsVersion2(context.TODO(), testclient.Client)
+				if err != nil {
+					Expect(err).ToNot(HaveOccurred())
+				}
+				if isV2 {
+					rootCgroupPath = "/rootfs/sys/fs/cgroup/cgroup.procs"
+				}
+				// Getting the list of PIDS that are running on the root cgroup.
+				output, err := nodes.ExecCommandOnNode(context.TODO(), []string{"cat", rootCgroupPath}, &node)
+				Expect(err).ToNot(HaveOccurred())
+				pids := strings.Split(output, "\n")
+				// Retrieving the processes represented by these PIDS.
+				output, err = nodes.ExecCommandOnNode(context.TODO(), append([]string{"ps", "-o", "cmd"}, pids...), &node)
+				Expect(err).ToNot(HaveOccurred())
```

Comment:
can we run a pipe to get the final result in one go?

Reply:
Unfortunately no. I learned it the hard way.

Reply:
and what's the problem? why doesn't it work?

Reply:
> That's because the underlying ExecPCommandOnPod function expect the [] string command to be array of argv. It takes commad[0] as the name of the cmd, and the latter slice as params. When I tried to change the line to `output, err := nodes.ExecCommandOnNode(context.TODO(), []string{"cat", rootCgroupPath, "|", "xargs", "ps", "-o", "cmd"}, &node)` and run the test this is what I get:
> 
> ```
>   [FAILED] Unexpected error:
>       <*fmt.wrapError | 0xc000242420>: 
>       failed to run command [cat /rootfs/sys/fs/cgroup/cgroup.procs | xargs ps -o cmd]: output "cat: invalid option -- 'o'\r\nTry 'cat --help' for more information.\r\n"; error ""; command terminated with exit code 1
>       {
>           msg: "failed to run command [cat /rootfs/sys/fs/cgroup/cgroup.procs | xargs ps -o cmd]: output \"cat: invalid option -- 'o'\\r\\nTry 'cat --help' for more information.\\r\\n\"; error \"\"; command terminated with exit code 1",
>           err: <exec.CodeExitError>{
>               Err: <*errors.errorString | 0xc0003085c0>{
>                   s: "command terminated with exit code 1",
>               },
>               Code: 1,
>           },
>       }
> ```

And what about something like
```
/bin/bash -c "/bin/cat $rootCgroupPath | /bin/xargs /bin/ps -o cmd"
```

Reply:
That sounds like a nice workaround indeed!

Reply:
@ffromani Updated the code according to your suggestion.

Reply:
cool, if you verified it works let's keep it

Reply:
@ffromani Yes, I have verified it (both cgroups v1 and v2).
BTW now I noticed, if we can use pipe with your workaround, maybe I also pipe a cmd to filter the kernel threads (`| grep -v "\["`) in the node instead of filtering the output.
WDYT? I will also add a more descriptive comment to explain the complicated command.

Reply:
note this is not a workaround but a fully legit way to run commands
that said yes, you can move the filtering in the pipeline, and please comment it.
The `ps` output includes both userspace processes and kernel threads, but kernel threads are always presented in `[square brackets]`.

Reply:
Done.